samples/DLLMethodCaller/FortuneTeller.cs

```diff
@@ -7,15 +7,23 @@ namespace DLLMethodCaller
 {
     class FortuneTeller
     {
+        private int total;
+        private int fortuneTellerFee = 1;
         public void AcceptMoney(int amount)
         {
-            // TODO
+            total += amount;
         }
 
         public string PredictFuture(DateTime birthday, bool isMarried)
         {
-            // TODO
-            return "";
+            total -= fortuneTellerFee;
+            if (total ==0 || isMarried)
+                return "You're broke!";
+            if (total < 0 )
+                return "You're in debt!";
```

Comment:
I think this case is not possible. Let's remove it and simplify. I think it would be easier to read this code if FortuneTeller takes all the money that it got no matter what. (e.g. reset total to zero each time you call PredictFuture).

Reply:
Cool. I was thinking if someone tries to PredictFuture (..) without depositing money.